### ISSUE TITLE
omake doesn't build on opam

### ISSUE THREAD

**Issue by c-cube** [ocaml-omake/omake#63]
On 4.03.0, opam 1.2.2, I get the following after `opam install omake`

```
### stderr ###
# [...]
# Use String.uncapitalize_ascii instead.
# File "omake_builtin_ocamldep.ml", line 36, characters 18-35:
# Warning 3: deprecated: String.capitalize
# Use String.capitalize_ascii instead.
# Updating the .config file.
# *** omake error:
#    File doc/OMakefile: line 227, characters 5-18
#    directory does not exist: /home/simon/.opam/4.03.0/build/omake.0.10.1/doc/txt
# Command exited with error: boot/omake '--dotomake' '.omake' '--force-dotomake' 'main' 'OCAML=ocaml'
# make: *** [Makefile:11: bootstrap] Error 1
```

**Comment by gerdstolpmann** [comment]
@rgrinberg I guess this is a problem with the source package. The file I originally distributed (projects.camlcity.org/projects/omake.html) contains the documentation, whereas the git tag does not.

**Comment by rgrinberg** [comment]
I'm not able to reproduce this issue on 4.03.0/1.2.2 omake seems to install just fine.

**Comment by rgrinberg** [comment]
I'm still wondering how this is only a problem for some of the users. @octachron seems to be experiencing this issue as well.

@gerdstolpmann would it be possible to include the doc. building  as part of the build  process? If not, then it would be great if it was optional. People should still be able to install omake straight from the git repo (through pinning)

**Comment by rgrinberg** [comment]
@octachron hinted at the root of the issue here.

The build in opam fails only when hevea is installed. It would be preferable not to build the docs automatically in this situation. opam expects to the docs to be buildable using the `build-doc:` set of commands in the opam file anyway.

**Comment by Armael** [comment]
I can reproduce the issue on my setup (and hevea is indeed installed). omake builds if I uninstall hevea.

**Comment by gerdstolpmann** [comment]
Sees to be fixed by commit 8e2371e4e85963ca2a5c304f8e23bd87856441b5
